Description: sftp_file: use Thread.daemon property instead of deprecated setDaemon
 The daemon property is available since at least Python 3.5.
Author: Simon Chopin <simon.chopin@canonical.com>
Bug: https://github.com/paramiko/paramiko/issues/1838
Forwarded: Not needed, already part of a bigger PR
Last-Update: 2022-01-26
--- a/paramiko/sftp_file.py
+++ b/paramiko/sftp_file.py
@@ -527,7 +527,7 @@
         self._prefetch_done = False
 
         t = threading.Thread(target=self._prefetch_thread, args=(chunks,))
-        t.setDaemon(True)
+        t.daemon = True
         t.start()
 
     def _prefetch_thread(self, chunks):
